ww3tools/modelSat_collocation.py

```diff
@@ -427,8 +435,12 @@
 fcy.close(); del fcy
 print(' Data Collocation/Matchups Ok.')
 
-# Quality Control
-ind=np.where( (fwhs>0.0) & (fwhs<20.0) & (fwwnd>0.0) & (fwwnd<60.0) & (fshs>0.0) & (fshs<20.0) & (fswnd>0.0) & (fswnd<60.0) )
+# Quality Control (yes or no)
+if qc=="yes":
```

Comment:
Thanks for adding this, I'll try to take a to do to move this to an input (yaml file or otherwise) level switch control.

Reply:
I'm still modifying modelSat_collocation using the configuration arguments from ww3tools.yaml.
Next version is coming soon and it will be the same as in ProcSat_Altimeter.py